skyproj/_skyproj.py

```diff
@@ -1091,7 +1114,8 @@ def draw_polygon_file(self, filename, reverse=True,
 
     @add_func_docstr(draw_hpxmap_docstr)
     def draw_hpxmap(self, hpxmap, nest=False, zoom=True, xsize=1000, vmin=None, vmax=None,
-                    rasterized=True, lon_range=None, lat_range=None, **kwargs):
+                    rasterized=True, lon_range=None, lat_range=None,
+                    log_scale=False, norm="linear", **kwargs):
```

Comment:
Do we need to track "log_scale" explicitly or can we determine it from the norm? What happens if the user provides norm="linear" and log_scale=True?

Reply:
Fixed!